charts/ui/values.yaml

```diff
@@ -29,3 +29,5 @@ resources: {}
 nodeSelector: {}
 
 tolerations: []
+
+podLabels: {}
```

Comment:
This hogs on to the bitnami naming convention. Other options would be `additionalLabels`, `labels` or something completely different?

Reply:
I think were are consistently inconsistent (like the rest of this space) let's stick with what's here

Reply:
I couldn't find any other place where we do pass labels yet, so maybe let's try to stick doing new things the "bitnami way" then?